js/src/3rdparty/sms-verification/index.js

```diff
@@ -15,17 +15,6 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 import { stringify } from 'querystring';
-import React from 'react';
-
-export const termsOfService = (
```

Comment:
Markdown should get compiled on build. What about [the template system currently in place](https://github.com/ethcore/parity/blob/6822433/js/src/modals/CreateAccount/recovery-page.ejs#L40-L47)? One you combine it with `dangerouslySetInnerHTML`.